styles.css

```diff
@@ -80,6 +84,59 @@ ul {
   color: #fff;
 }
 
+#mobile-navigation.active {
+  left: 0;
+}
+
+#mobile-navigation {
+  position: fixed;
+  background-color: #3c3a39;
+  left: 100%;
+  top: -25px;
+  width: 100%;
+  padding: 140px 20px 0;
+  height: 100%;
```

Comment:
- [ ] please add the correct padding or margin top value by decreasing it to match the given Figma design.

| Given Design  | Your Design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/68796232/210449022-9ac8f98d-0bbe-4e69-9815-806646503d4a.png)  | ![image](https://user-images.githubusercontent.com/68796232/210449060-2af8b21f-d521-4842-9d63-1667a3467627.png) |